frontend/src/components/common/form/RadioButtonGroup.tsx

```diff
@@ -90,8 +90,8 @@ export const RadioButtonGroup: React.FC<RadioGroupProps> = ({
             ></Form.Check>
           ))}
         </div>
+        <DisplayError field={field} />
```

Comment:
needs to be a sibling of the input to display properly because bootstrap is weird at times.